huxley/www/js/components/AdvisorFeedbackView.js

```diff
@@ -0,0 +1,195 @@
+/**
+* Copyright (c) 2011-2015 Berkeley Model United Nations. All rights reserved.
+* Use of this source code is governed by a BSD License (see LICENSE).
++*/
+
+'use strict';
+
+var React = require('react');
+var ReactRouter = require('react-router');
+
+var AssignmentActions = require('actions/AssignmentActions');
+var AssignmentStore = require('stores/AssignmentStore');
+var Button = require('components/Button');
+var CommitteeStore = require('stores/CommitteeStore');
+var ConferenceContext = require('components/ConferenceContext');
+var CountryStore = require('stores/CountryStore');
+var CurrentUserStore = require('stores/CurrentUserStore');
+var DelegateStore = require('stores/DelegateStore');
+var InnerView = require('components/InnerView');
+var ServerAPI = require('lib/ServerAPI');
+
+var AdvisorFeedbackView = React.createClass({
+  mixins: [
+    ReactRouter.History,
+  ],
+
+  contextTypes: {
+    conference: React.PropTypes.shape(ConferenceContext)
+  },
+
+  getInitialState: function() {
+    var schoolID = CurrentUserStore.getCurrentUser().school.id;
+    var delegates = DelegateStore.getSchoolDelegates(schoolID);
+    var assigned = this.prepareAssignedDelegates(delegates);
```

Comment:
This should instead be called `feedback` or something along those lines because you are mapping assignment ids to feedback on the delegates. Additionally, `prepareFeedback` is probably a more suitable name